src/applications/disability-benefits/all-claims/components/IntroductionPage.jsx

```diff
@@ -113,10 +115,35 @@ class IntroductionPage extends React.Component {
           <ol>
             <li className="process-step list-one">
               <h3 className="vads-u-font-size--h4">Prepare</h3>
-              <p>
-                When you file a disability claim, you’ll have a chance to
-                provide evidence to support your claim. Evidence could include:
-              </p>
+              {!isShowBDDSHA && (
+                <p data-testid="process-step1-prepare">
+                  When you file a disability claim, you’ll have a chance to
+                  provide evidence to support your claim. Evidence could
+                  include:
+                </p>
+              )}
+              {isShowBDDSHA && (
+                <>
+                  <p data-testid="process-step1-prepare">
+                    When you file a BDD claim online, we'll ask you to upload
+                    this required form:{' '}
+                    <a href={DBQ_URL} target="_blank" rel="noreferrer">
+                      Separation Health Assessment - Part A Self-Assessment
+                    </a>
+                    . We recommend you download and fill out this form on a
+                    desktop computer or laptop. Then return to this page to
+                    start the application process.
+                  </p>
+                  <p>
+                    <strong>Note:</strong> We estimate that it will take you at
+                    least 30 minutes to complete this form.
+                  </p>
+                  <p>
+                    You'll also have a chance to provide this type of evidence
```

Comment:
Not sure why eslint isn't catching these... the single quote (`'`) should be replaced by a typographical apostrophe (`’`)